Mathlib/LinearAlgebra/SesquilinearForm.lean

```diff
@@ -386,7 +390,7 @@ theorem orthogonal_span_singleton_eq_to_lin_ker {B : V →ₗ[K] V →ₛₗ[J]
 
 -- todo: Generalize this to sesquilinear maps
 theorem span_singleton_sup_orthogonal_eq_top {B : V →ₗ[K] V →ₗ[K] K} {x : V} (hx : ¬B.IsOrtho x x) :
-    (K ∙ x) ⊔ @Submodule.orthogonalBilin _ _ _ _ _ _ (_) _ _ (K ∙ x) B = ⊤ := by
+    (K ∙ x) ⊔ @Submodule.orthogonalBilin _ _ _ _ _ _ _ _ _ (_) _ _ (K ∙ x) B = ⊤ := by
   rw [orthogonal_span_singleton_eq_to_lin_ker]
```

Comment:
It would be nice to get rid of the string of underscores here as well

Reply:
Done, thanks.